diff --git a/src/core/ext/transport/chaotic_good/server_transport.h b/src/core/ext/transport/chaotic_good/server_transport.h
--- a/src/core/ext/transport/chaotic_good/server_transport.h
+++ b/src/core/ext/transport/chaotic_good/server_transport.h
@@ -28,7 +28,6 @@
 #include <iostream>
 #include <map>
 #include <memory>
-#include <optional>
 #include <string>
 #include <tuple>
 #include <type_traits>
diff --git a/src/core/ext/transport/chaotic_good_legacy/server_transport.h b/src/core/ext/transport/chaotic_good_legacy/server_transport.h
--- a/src/core/ext/transport/chaotic_good_legacy/server_transport.h
+++ b/src/core/ext/transport/chaotic_good_legacy/server_transport.h
@@ -28,7 +28,6 @@
 #include <iostream>
 #include <map>
 #include <memory>
-#include <optional>
 #include <string>
 #include <tuple>
 #include <type_traits>
diff --git a/src/core/lib/security/credentials/tls/grpc_tls_crl_provider.h b/src/core/lib/security/credentials/tls/grpc_tls_crl_provider.h
--- a/src/core/lib/security/credentials/tls/grpc_tls_crl_provider.h
+++ b/src/core/lib/security/credentials/tls/grpc_tls_crl_provider.h
@@ -27,6 +27,5 @@
 #include <chrono>
 #include <functional>
 #include <memory>
-#include <optional>
 #include <string>
 #include <utility>
